codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
+  branch: dev
+
+coverage:
+  status:
+    project:
+      default:
+        # minimum coverage ratio that the commit must meet to be considered a success
+        target: 40%
```

Comment:
Is this based in the current coverage on the project?

Reply:
Yes, I adapted the values to the results of the current coverage report.